.babelrc

```diff
@@ -7,5 +7,19 @@
       }
     ],
     "@babel/preset-react"
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-transform-react-jsx",
+      {
+        "pragmaFrag": "React.Fragment" // default is React.Fragment
```

Comment:
If this is the default, can you omit the options block?

Reply:
No, actually that's wrong. I tried the default and it just leaves the bare `Fragment`, which is what was breaking in the first place.

Reply:
Maybe delete or clarify the comment, then; but very minor. 👍